### PR TITLE
33 feature see all moderators chekurkov

### PR DESCRIPTION
**Pull request by Dmitry528**
Sign in like admin and click on modetarors

### CODE REVIEW

File: src/components/dashboard/app-sidebar.vue [hunk 1]
```diff
@@ -45,7 +45,7 @@
                 <li class="navList-heading">Tools</li>
                 <li>
                     <ul class="subList subList">
-                        <li class="subList-item">Moderators</li>
+                        <li class="subList-item"><a href="http://localhost:8081/moderators">Moderators</a></li>
```
Comment (AndriiDruzhytskyi) [inline src/components/dashboard/app-sidebar.vue]: Please don`t hardcode url address.

File: src/pages/app-moderators.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+              rows: []
+          }
+        },
+        methods: {
+          
+        },
+        created() {
+            ModeratorService.getAllModerators()
+            .then((moderators) => {
+                console.log(moderators);
```
Comment (AndriiDruzhytskyi) [inline src/pages/app-moderators.vue]: Do we need this debug output in dev?

**Review by andrii-kondratiuk — APPROVED** [review]

**Review by AndriiDruzhytskyi — APPROVED** [review]

**Review by marrybbb — APPROVED** [review]

File: src/components/dashboard/app-sidebar.vue (hunk 1 above)
Comment (Dmitry528) [reply src/components/dashboard/app-sidebar.vue]: I binded in

File: src/pages/app-moderators.vue (hunk 2 above)
Comment (Dmitry528) [reply src/pages/app-moderators.vue]: I delete console log

**Pull request merged**